integration_tests/tests/forseti/controls/logs-debug_level.rb

```diff
@@ -20,9 +20,8 @@
 random_string = SecureRandom.uuid.gsub!('-', '')[0..10]
 
 control "logs-debug-level" do
-
   # Get the instance id
-  instance_describe_cmd = command("gcloud compute instances describe #{forseti_server_vm_name} --zone us-central1-c --format=json")
+  instance_describe_cmd = command("sudo gcloud compute instances describe #{forseti_server_vm_name} --zone us-central1-c --format=json")
```

Comment:
I ran into an error with this test locally. I got a permissions error when running it on the server via SSH. Running the command with sudo resolved that issue.